sdcm/tester.py

```diff
@@ -1132,8 +1133,10 @@ def create_table(self, name, key_type="varchar",  # pylint: disable=too-many-arg
             # will default to lz4 compression
             query += ' AND compression = {}'
 
-        if compaction is not None:
-            query += " AND compaction={'class': '%s'}" % compaction
+        if compaction is not None or sstable_size:
+            compaction = compaction or self.params.get('compaction_strategy', default=CompactionStrategy.SIZE_TIERED.value)
```

Comment:
is SizeTiered the default in scylla ? i.e. the default we had in this code until now ?
or this PR change this function believer ?

Reply:
SizeTiered is scylla's default. it may change in the future to ICS in enterprise.